### PR TITLE
add Keppen defense

### PR DESCRIPTION
**Pull request by dougwoods3**

### CODE REVIEW

File: Events/index.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ layout: default
 author: Doug, Jackson, & Tony
 ---
 
+# Upcoming Defenses
+* [Jackson Keppen](https://rtrp.github.io/osu-transport/users/keppenj/)
+- Date: May 14, 2018
+- Time: 2 pm
+- Location: Learning Innovation Center 360
```
Comment (albeanth) [inline Events/index.md]: is Jackson a masters student?

**Comment by albeanth** [comment]
:shipit:

**Comment by dougwoods3** [comment]
@rtrp/admins Can you guys check the TexGraphics.tex file? It should be stand alone.

File: users/woodsdou/index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Doug Woods
 
 <img src="{{ site.url }}users/woodsdou/images/Doug%20Woods%20Three%205%20to%207%20crop.JPG" width="200">
 
-I am a Ph.D. candidate in Nuclear Engineering in the [School of Nuclear Science and Engineering](https://ne.oregonstate.edu) at Oregon State University. My major advisor is [Todd Palmer](https://rtrp.github.io/osu-transport/palmerts/). I am also a licenced reactor operator for the [Oregon State TRIGA Reactor](http://radiationcenter.oregonstate.edu/oregon-state-triga-reactor-0) and the Oregon State [ANS & HPS](http://groups.engr.oregonstate.edu/ANS/) vice president.
+I am a Ph.D. candidate in Nuclear Engineering in the [School of Nuclear Science and Engineering](https://ne.oregonstate.edu) at Oregon State University. My major advisor is [Todd Palmer](https://rtrp.github.io/osu-transport/palmerts/). I am also a licensed reactor operator for the [Oregon State TRIGA Reactor](http://radiationcenter.oregonstate.edu/oregon-state-triga-reactor-0) and the Oregon State [ANS & HPS](http://groups.engr.oregonstate.edu/ANS/) vice president.
```
Comment (harterj) [inline users/woodsdou/index.md]: You are no longer a PhD candidate! You're the real deal!

**Comment by harterj** [comment]
@dougwoods3 what do you want done with this? Is TexGraphics in that zip file? Thanks for updating the Gemfile.

**Comment by harterj** [comment]
:shipit: I downloaded the zip for TeXGraphics and it all seemed to check out. I approve -- Doug push when ready.

**Pull request merged**